File: src/p0771/golang/solution.go
func numJewelsInStones(J string, S string) (result int) {
	dict := make(map[int32]int)
	for _, k := range J {
		dict[k] = 1
	}
	for _, k := range S {
		result += dict[k]
	}
	return result
}

File: src/p0771/python/solution.py
class Solution:
    def numJewelsInStones(self, j: str, s: str) -> int:
        result = 0
        table = [0] * 256
        for k in j:
            table[ord(k)] = 1
        for k in s:
            result += table[ord(k)]
        return result
